givitsite/accounts/models.py

```diff
@@ -1,5 +1,31 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+MEMBERSHIP_TYPE = [
+    ('friend','friend'),
+    ('coordinator','coordinator'),
+]
 
+DEGREE = [
+    ('BA', 'BA'),
+    ('BSC', 'BSC'),
+    ('MA', 'MA'),
+    ('MSC', 'MSC'),
+    ('PHD', 'PHD'),
+]
 
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    
+    confirmed = models.BooleanField("confirmed", default=False)
+    student_id = models.IntegerField()
+    phone_number = models.IntegerField()
```

Comment:
phone number starting with zero, I think that integer isn't keeping on it

Reply:
https://github.com/google/libphonenumber/blob/master/FALSEHOODS.md

Reply:
Hi as Lior mentioned I wanted first to modify the mistake from #37 PR, so all data will be save when register and we could finish the coordinator part in the project. Now I will add validations and beautify the page design.
Moreover, as you (barak) note me in previous PR I tried to keep it small and focus PR

Reply:
ok. lets have a ticket about this, and move on